docs/docs/configuration/sessions.md

```diff
@@ -52,6 +52,33 @@ in redis via the `SETEX` command.
 Encrypting every session uniquely protects the refresh/access/id tokens stored in the session from
 disclosure.
 
+Additionally the browser only has to send a short Cookie with every request and not the whole JWT, which can get quite big.
+
+Two settings are used to configure the OAuth2-Proxy cookie lifetime:
+
+    --cookie-refresh duration   refresh the cookie after this duration; 0 to disable
+    --cookie-expire duration    expire timeframe for cookie     168h0m0s
+
+The "cookie-expire" value should be equal to the lifetime of the Refresh-Token that is issued by the OAuth2 authorization server.
+If it expires earlier and is deleted by the browser, oauth2-proxy cannot find the stored Refresh-Tokens in Redis and thus cannot start
+the refresh flow to get new Access-Tokens. If it is longer, it might be that the old Refresh-Token will be found in Redis but has already
+expired.
+
+The "cookie-refresh" value controls when OAuth2-Proxy tries to refresh an Access-Token. If it is set to "0", the
```

Comment:
Elsewhere on this page it's referred to as OAuth2 Proxy (although there's some inconsistency in the docs as it's also refered to as `oauth2-proxy` on other pages).

Reply:
Changed to "OAuth2 Proxy". 
Is the use of "Access-Token" instead of "access token" OK when referring to the terms of the standard and not just any access tokens? Feel free to add changes if you want.